fetch-in-react-practice/client/components/app.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PageTitle from './page-title';
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: []
+    };
+    this.addTodo = this.addTodo.bind(this);
+    this.toggleCompleted = this.toggleCompleted.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('/api/todos')
+      .then(response => response.json())
+      .then(data => {
+        const todoList = [];
+        for (let i = 0; i < data.length; i++) {
+          todoList.push(data[i]);
+        }
```

Comment:
No need for a loop here, you can directly set `data` to the `todos` property in state with `setState`